src/globus_sdk/services/transfer/data.py

```diff
@@ -212,7 +214,7 @@ def add_item(
         recursive=False,
         external_checksum=None,
         checksum_algorithm=None,
-        **params,
+        additional_fields: Optional[Dict[str, Any]] = None,
     ):
```

Comment:
Unlike the top-level extra parameters, this one is more useful to keep.

I'm still ruminating about a better name for it, but nothing comes to mind as obviously better.

Reply:
As we discussed, this name is actually pretty good. Marking this resolved.